common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -112,6 +112,7 @@ class VideoModule(VideoFields, VideoTranscriptsMixin, VideoStudentViewHandlers,
     #TODO: For each of the following, ensure that any generated html is properly escaped.
     js = {
         'js': [
+            resource_string(module, 'js/src/time.js'),
```

Comment:
@pomegranited Weird. Really weird. I haven't read through all the comments on the ticket (if there are any), but is there an explanation to that? Just "I've added it and it stopped failed the test" is enough, but it would be nive to have a deeper understanding.